Cirrus: Upload swagger YAML in every context

cevich opened the pull request:
The podman documentation site uses javascript to display
API documentation at:

http://docs.podman.io/en/latest/Reference.html

As input, the javascript sources from a CORS-enabled Google Cloud
Storage object.  This commit ensures the storage object is present and
updated for every Cirrus-CI execution context: Tags, Branches, and PRs.

As of this commit, the documentation site only utilizes the object
uploaded by the Cirrus-CI run on the `master` branch:
`swagger-master.yaml`.  The file produced and uploaded due to a PR is
intended for testing purposes: Confirm it's generation and uploading are
both functional.

edsantiago commented:
I see tests failing, and `TODO` in the diffs. @cevich can you ping me when this is truly ready for review?

cevich commented:
@edsantiago oh I'm sorry, did Github notify you?  I thought it would not since the PR is still set as a 'draft'.

In any case, it's ready now for an *initial* peek, and [it's working as expected for a PR.](https://cirrus-ci.com/task/6367810920120320?command=main#L66-L73).  Note: There's still a big-bad ***DO NOT MERGE*** commit (and I'll leave the PR in 'draft' state) as I need to grab (and test) images after https://github.com/containers/automation_images/pull/47 is working.

cevich commented:
...and apparently a `setup_environment.sh` conflict as well.

cevich commented:
Rebased with some minor changes.  Added temporary hard-coding of $CTR_FQIN for the swagger task, due to current exceptional churn in VM images.  Noted this in a `TODO` comment.

cevich commented:
(Woopsiedoodle...bad copy-paste of gcsupld image suffix value.)

cevich commented:
@baude @edsantiago PTAL, this should be ready to go.

baude commented:
/approve

baude commented:
/lgtm

cevich commented:
Ref: [Post-merge, successfully uploaded new swagger yaml.](https://cirrus-ci.com/task/6445762227732480)
